plugin.py

```diff
@@ -64,6 +64,22 @@ def logAndShowMessage(msg, additional_logs=None):
     sublime.active_window().status_message(msg)
 
 
+def getGlobalSnippetDir() -> str:
+    """
+    Returns default global directory for user's snippets.
+
+    Uses same logic and paths as vetur.
+    See https://github.com/vuejs/vetur/blob/master/client/userSnippetDir.ts
+    """
+    appName = 'Code'
+    if sublime.platform() == 'windows':
+        return os.path.expandvars('%%APPDATA%%\\{}\\User\\snippets\\vetur').format(appName)
+    elif sublime.platform() == 'osx':
+        return os.path.expanduser('~/Library/Application Support/{}/User/snippets/vetur').format(appName)
+    else:
+        return os.path.expanduser('~/.config/{}/User/snippets/vetur').format(appName)
```

Comment:
You could also use `os.path.join(sublime.package_path(), '..', '..', 'Code', 'User', 'snippets', 'vetur')` and get rid of the if-elses.

Reply:
Hmm, nope, on windows I get C:\\Users\\rachwols\\AppData\\Roaming\\Code\\User\\snippets\\vetur unfortunately.

Reply:
But `sublime.packages_path()` would be wrong to use if it's a portable Sublime, right?

Reply:
Just tried with a portable installed, and you're correct:

```
>>> sublime.packages_path()
'C:\\Users\\rachwols\\Downloads\\SublimeText\\Data\\Packages'
```